Doc/library/dis.rst

```diff
@@ -293,17 +293,14 @@ details of bytecode instructions as :class:`Instruction` instances:
 
    .. data:: argval
 
-      resolved arg value (if known), otherwise ``dis.UNKNOWN``
+      resolved arg value (if any), otherwise ``None``
 
-      .. versionchanged:: 3.11
-         Unknown values are now set to an appropriate sentinel.
 
    .. data:: argrepr
 
-      human readable description of operation argument (if known).
+      human readable description of operation argument (if any),
```

Comment:
I like how you kept it ambiguous whether "if any" refers to the argument or to the description. :-)